platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -538,6 +599,16 @@ pre code {
   padding-left: 2px;
 }
 
+.show-on-github a {
+  padding: 0px 5px;
+  background: rgba(56, 135, 190, 0.12);
```

Comment:
This makes the “View Source on GitHub” link much more prominent than before. While I like the treatment, it may cause these links to call too much attention to themselves. Unlike with GL JS, these links only go to the declaration and documentation comment for the symbol – the exact same thing as the content right above the link, except set in a monospaced font and wrapped at 80 columns. As far as I can tell, the main use case for these links is for code contributors to easily get to the relevant header to correct typos.